<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SdlQtRGBClass</class>
 <widget class="QWidget" name="SdlQtRGBClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SdlQtRGB</string>
  </property>
  <widget class="QLabel" name="label_video1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>331</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 0);</string>
   </property>
   <property name="text">
    <string>video1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_video2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>331</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 255);</string>
   </property>
   <property name="text">
    <string>video2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_open1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_frame1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>YUV420p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ARGB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BGRA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RGBA</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinBox_width1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>400</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_height1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>430</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>300</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_fps1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>470</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>120</number>
   </property>
   <property name="value">
    <number>25</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_open2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_frame2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>350</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>YUV420p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ARGB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BGRA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RGBA</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinBox_fps2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>470</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>120</number>
   </property>
   <property name="value">
    <number>25</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_width2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>390</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>400</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_height2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>430</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>300</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_fps1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Fps:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_fps2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>Fps:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sdlqtrgb.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_open1</sender>
   <signal>clicked()</signal>
   <receiver>SdlQtRGBClass</receiver>
   <slot>Open1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_open2</sender>
   <signal>clicked()</signal>
   <receiver>SdlQtRGBClass</receiver>
   <slot>Open2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>636</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Open1()</slot>
  <slot>Open2()</slot>
 </slots>
</ui>
